src/Map.php

```diff
@@ -271,7 +362,7 @@ public function hasValue($value): bool
      */
     public function count(): int
     {
-        return count($this->pairs);
+        return count($this->pairRefs);
```

Comment:
count() doesn't count recursively i think so because we can have multiple values in a pairRef index couldn't this be inaccurate?